README.md

```diff
@@ -55,6 +55,8 @@ npx docusaurus-pdf from-build-config
 - Mandatory: `dirPath` which points to the build directory created with `docusaurus build`.
 - Mandatory: `firstDocPagePath` is the URL path segment (without `baseUrl`) of your first docs page you whish to have included in the PDF.
 - Optional: If you have a `baseUrl` configured in your `docusaurus.config.js` then pass this value as `baseUrl`.
+- Optional: You can specify larger/smaller margins, e.g. `--margin "1cm 1.5cm 1cm 2cm"` (order top right bottom left, like the css margin field).
```

Comment:
I tried this but unfortunately this doesn't work for me:

```
   ~/Dow/tm/docusaurus-pdf/t/test-website     add-puppeteer-opts !1  node ../../bin/index.js from-build-config --margin "1cm 1.5cm 1cm 2cm"
Error: Was expecting exactly 4 margin specifiers, instead got 5. Margin specifier was "1cm 1.5cm 1cm 2cm".
```

There seems to be a bug if a decimal number is used 🙃

Reply:
Good catch! I didn't even think of people using 1.5 cm, even though it's something I always do with LaTeX.

Reply:
Hey @aloisklink! Any progress on this? Let me know if I can help you bringing this PR into production!